packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -105,11 +105,16 @@ export async function fetchHelper(
         const response = fetchResponse as any as Response;
         // Let's assume we can retry.
         if (!response) {
-            throwOdspNetworkError(`No response from the server`, fetchIncorrectResponse);
+            throwOdspNetworkError(`noResponseFromTheServer`, fetchIncorrectResponse);
         }
         if (!response.ok || response.status < 200 || response.status >= 300) {
             throwOdspNetworkError(
-                `Error ${response.status}`, response.status, response, await response.text());
+                "errorResponseNotOk",
```

Comment:
I'm not sure it will make it better. I understand why we are trying not to merge strings, but it's super useful today that one can look at eventName / reason / error properties across 95% of various queries and make sense of what's going on without need to look at the code to understand what properties to add to query to make sense of it.
I'm prefer to have that flow for simplicity (humans). And in places where it's not good enough (automated queries), keep adding specialized properties without removing human-readable messages.

Reply:
Makes sense. I will add back the status here.